vendor/bat-native-ledger/src/bat/ledger/internal/bat_contribution.cc

```diff
@@ -1032,42 +1033,50 @@ void BatContribution::GetContributeWinners(
     const unsigned int ballots,
     const std::string& viewing_id,
     const braveledger_bat_helper::PublisherList& list) {
-  unsigned int total_votes = 0;
-  std::vector<unsigned int> votes;
-  braveledger_bat_helper::Winners res;
+  braveledger_bat_helper::Winners winners =
+      GetStatisticalVotingWinners(ballots, list);
+  VotePublishers(winners, viewing_id);
+}
 
-  for (auto &item : list) {
-    if (item.percent_ <= 0) {
+bool BatContribution::GetStatisticalVotingWinner(
+    double dart,
+    const braveledger_bat_helper::PublisherList& list,
+    braveledger_bat_helper::WINNERS_ST* winner) {
```

Comment:
could this be ref and not a pointer?

Reply:
Happy to do it but I think @bridiver had mentioned to me before to prefer to pass params like this as pointers, so it's obvious from the caller's point of view what's happening.

Reply:
sounds good, yeah I think linter would also complain about the reference if I remember correctly